docs/Getting Started/Arch Linux/Arch Linux Root on ZFS.rst

```diff
@@ -396,7 +405,7 @@ Create Root and Boot Pools
         -O xattr=sa \
         -O mountpoint=/ \
         -R $INST_MNT \
-        -O encryption=aes-256-gcm \
+        -O encryption=on \
```

Comment:
Is this OpenZFS 2.0 or an older version?

Reply:
No, latest 2.0.3.

Reply:
Defaults to aes-256-gcm, as you know.

Reply:
To be clear, if it's OpenZFS 2.0, then on is the same as aes-256-gcm, so "on" is good. If it's older, then you should use "aes-256-gcm" explicitly. If it's OpenZFS 2.0, can you just confirm for me.

Also, can you squash this (at least somewhat)? Then it should be good to merge.